cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -1007,12 +1054,14 @@ instance (Condense (HeaderHash blk), LedgerSupportsProtocol blk)
     ChainDB.TrySwitchToAFork pt _ ->
       mkObject [ "kind" .= String "TraceAddBlockEvent.TrySwitchToAFork"
                , "block" .= toObject verb pt ]
-    ChainDB.AddedToCurrentChain _ _ c ->
+    ChainDB.AddedToCurrentChain _ base extended  ->
       mkObject [ "kind" .= String "TraceAddBlockEvent.AddedToCurrentChain"
-               , "newtip" .= showPoint verb (AF.headPoint c) ]
-    ChainDB.SwitchedToAFork _ _ c ->
+               , "newtip" .= showPoint verb (AF.headPoint extended)
+               , "headers" .= toJSON (toObject verb <$> addedHdrsNewChain base extended) ]
+    ChainDB.SwitchedToAFork _ old new ->
       mkObject [ "kind" .= String "TraceAddBlockEvent.SwitchedToAFork"
-               , "newtip" .= showPoint verb (AF.headPoint c) ]
+               , "newtip" .= showPoint verb (AF.headPoint new)
+               , "headers" .= toJSON (toObject verb <$> addedHdrsNewChain old new) ]
```

Comment:
Don't we only want to include this huge amount of extra detail if the verbosity (`verb` var) is at a higher level? Otherwise this is now going to be very verbose isn't it?

The `<$>` here is just list `map` right? Can we just use `map`?

Reply:
Done on both counts.